.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,14 @@
+name: Deploy to dev
```

Comment:
Minor, but the indentation seems off in this file (4 spaces) - other files seem to use 2 spaces so let's be consistent